code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -223,68 +201,61 @@
 	desc = "A small box capable of holding seven shotgun shells."
 	icon_state = "buckshotbox"
 
-/obj/item/storage/box/buck/New()
-	..()
-	for(var/i in 1 to 7)
+/obj/item/storage/box/buck/populate_contents()
+	for(var/I in 1 to 7)
 		new /obj/item/ammo_casing/shotgun/buckshot(src)
 
 /obj/item/storage/box/dragonsbreath
 	name = "Ammunition Box (Dragonsbreath)"
 	desc = "A small box capable of holding seven shotgun shells."
 	icon_state = "dragonsbreathbox"
 
-/obj/item/storage/box/dragonsbreath/New()
-	..()
-	for(var/i in 1 to 7)
+/obj/item/storage/box/dragonsbreath/populate_contents()
+	for(var/I in 1 to 7)
 		new /obj/item/ammo_casing/shotgun/incendiary/dragonsbreath(src)
 
 /obj/item/storage/box/stun
 	name = "Ammunition Box (Stun shells)"
 	desc = "A small box capable of holding seven shotgun shells."
 	icon_state = "stunbox"
 
-/obj/item/storage/box/stun/New()
-	..()
-	for(var/i in 1 to 7)
+/obj/item/storage/box/stun/populate_contents()
+	for(var/I in 1 to 7)
 		new /obj/item/ammo_casing/shotgun/stunslug(src)
 
 /obj/item/storage/box/beanbag
 	name = "Ammunition Box (Beanbag shells)"
 	desc = "A small box capable of holding seven shotgun shells."
 	icon_state = "beanbagbox"
 
-/obj/item/storage/box/beanbag/New()
-	..()
-	for(var/i in 1 to 7)
+/obj/item/storage/box/beanbag/populate_contents()
+	for(var/I in 1 to 7)
 		new /obj/item/ammo_casing/shotgun/beanbag(src)
 
 /obj/item/storage/box/rubbershot
 	name = "Ammunition Box (Rubbershot shells)"
 	desc = "A small box capable of holding seven shotgun shells."
 	icon_state = "rubbershotbox"
 
-/obj/item/storage/box/rubbershot/New()
-	..()
-	for(var/i in 1 to 7)
+/obj/item/storage/box/rubbershot/populate_contents()
+	for(var/I in 1 to 7)
 		new /obj/item/ammo_casing/shotgun/rubbershot(src)
 
 /obj/item/storage/box/tranquilizer
 	name = "Ammunition Box (Tranquilizer darts)"
 	desc = "A small box capable of holding seven shotgun shells."
 	icon_state = "tranqbox"
 
-/obj/item/storage/box/tranquilizer/New()
-	..()
-	for(var/i in 1 to 7)
+/obj/item/storage/box/tranquilizer/populate_contents()
+	for(var/I in 1 to 7)
 		new /obj/item/ammo_casing/shotgun/tranquilizer(src)
 
 /obj/item/storage/box/flashbangs
 	name = "box of flashbangs (WARNING)"
 	desc = "<B>WARNING: These devices are extremely dangerous and can cause blindness or deafness in repeated use.</B>"
 	icon_state = "flashbang"
 
-/obj/item/storage/box/flashbangs/New()
-	..()
+/obj/item/storage/box/flashbangs/populate_contents()
 	new /obj/item/grenade/flashbang(src)
```

Comment:
Nitpick: I feel like these should also become `for(var/i in 1 to X)` loops

Reply:
Yeah I wasn't sure about what level it should switch to using a loop instead.
If it's spawning 20 items then it should definitely be a loop, but if it's only three or four... *shrug

Reply:
I mean its used in some places that spawn 4, so I would probably convert it. Also makes it very obvious to anyone who reads the code

Reply:
Alright, I'll change it for anything above four items. Gimme a sec. 👍